pyproject.toml

```
@@ -3,15 +3,79 @@ requires = [
     "setuptools",
     "packaging",
     "wheel",
-    "cython>=0.29.20; python_version>='3.10'",
-    "cython>=0.29.20,<3.0.0; python_version<='3.9'",
-    # See https://numpy.org/doc/stable/user/depending_on_numpy.html for
-    # the recommended way to build against numpy's C API:
+    "cython>=0.29.20",
     "numpy>=2.0.0",
     "scipy>=1.9",
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "qutip"
+dynamic = ["version"]
+description = "QuTiP: The Quantum Toolbox in Python"
+readme = "README.md"
+requires-python = ">=3.10"
+license = {text = "BSD-3-Clause"}
```

```suggestion
license = "BSD-3-Clause"
```
Setting `text = ...` specifies that the value is the full license document. We want exactly the suggestion above. Does the suggestion not work for some reason?

---

Ah, that is annoying. Maybe let's go back to `license = {file = ...}` then as more accurate than `text=`?